client-app/src/Components/Programs.tsx

```diff
@@ -20,7 +20,7 @@ function Programs() {
         const fetchPrograms = async () => {
             try {
                 const response = await axios.get<Program[]>(
-                    `${process.env.REACT_APP_BACKEND_API_BASE_URL}program`,
+                    `${process.env.REACT_APP_BACKEND_API_BASE_URL}/program`,
```

Comment:
@miabu-pashh Why did you bring up this change. I already mentioned to follow the existing convention and remove the `'/'` before `program`.

Please update the backend api variable in .env file to `REACT_APP_BACKEND_API_BASE_URL="http://localhost:4010/"`
There is already a leading '/' in this file, so you do not have to put it here.